rust-webvr-api/src/vr_display.rs

```diff
@@ -13,6 +13,9 @@ pub trait VRDisplay: Send + Sync {
     /// Returns the current display data.
     fn data(&self) -> VRDisplayData;
 
+    /// Returns gamepads attached to this display
+    fn fetch_gamepads(&mut self) -> Result<Vec<VRGamepadPtr>, String>;
```

Comment:
Any chance of a spec link? (Probably not, but you never know.)

Reply:
This isn't a specced method, I need it here to be able to safely fetch gamepads on the servo side without having to further complicate the unsafe code dealing with display pointers.